tests/test_culling.py

```diff
@@ -353,7 +396,13 @@ def test_unknown_host_is_not_removed(
     assert retrieved_host.reporter is None
 
     threadctx.request_id = UNKNOWN_REQUEST_ID_VALUE
-    host_reaper_run(inventory_config, mock.Mock(), db.session, event_producer_mock)
+    host_reaper_run(
+        inventory_config,
+        mock.Mock(),
+        db.session,
+        event_producer_mock,
+        shutdown_handler=mock.Mock(**{"shut_down.side_effect": (False)}),
```

Comment:
Here too a _return_value_ would be more appropriate.

```suggestion
        shutdown_handler=mock.Mock(**{"shut_down.return_value": False}),
```